src/towncrier/test/test_packaging.py

```diff
@@ -1,22 +1,16 @@
 # Copyright (c) Amber Brown, 2015
 # See LICENSE for details.
 
-from incremental import Version
 from twisted.trial.unittest import TestCase
 
-from towncrier._version import _hatchling_version
+import towncrier
 
 
 class TestPackaging(TestCase):
-    def test_version_warning(self):
+    def no_version_attr(self):
```

Comment:
I don't think this runs if it doesn't have a `test_` prefix.

Reply:
changed test to check for deprecationwarning instead #627